dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -138,7 +138,8 @@ public class ConnectorArguments extends DefaultArguments {
     private final OptionSpec<String> optionOracleSID = parser.accepts(OPT_ORACLE_SID, "SID name for oracle").withRequiredArg().describedAs("orcl").ofType(String.class);
     private final OptionSpec<String> optionConfiguration = parser.accepts("config", "Configuration for DB connector").withRequiredArg().ofType(String.class).withValuesSeparatedBy(';').describedAs("key=val;key1=val1");
     // private final OptionSpec<String> optionDatabase = parser.accepts("database", "database (can be repeated; all if not specified)").withRequiredArg().describedAs("my_dbname").withValuesSeparatedBy(',');
-    private final OptionSpec<File> optionOutput = parser.accepts("output", "Output file").withRequiredArg().ofType(File.class).describedAs("cw-dump.zip");
+    private final OptionSpec<File> optionOutput = parser.accepts("output", "Output file").withOptionalArg().ofType(File.class).describedAs("cw-dump.zip");
+    private final OptionSpec<String> optionOutputDir = parser.accepts("output-dir", "Output directory").withOptionalArg().ofType(String.class).defaultsTo("").describedAs("/home/user/...");
```

Comment:
1. I would recommend using `withRequiredArg` instead of `withOptionalArg` (similarly to the option above).
2. I would suggest to change the description from "Output directory" to "Output directory (cannot be used together with --output)".
3. nit: In my opinion, the description should not contain the ellipsis (`...`), but rather should be an example of a valid path, e.g. `/home/user/my-dump-dir`.

Reply:
I would suggest to use even more elaborate description, because "Output directory" seems confusing when compared with "Output file". For example:

"Output directory where the zip file with the default filename should be created (cannot be used together with --output option). If you want to specify the filename, use --output option instead.".